tests/conftest_test.py

```diff
@@ -0,0 +1,45 @@
+import ast
+import tokenize
+from io import StringIO
+
+import pytest
+
+from pandas_dev_flaker.__main__ import run
+
+
+def results(s):
+    return {
+        "{}:{}: {}".format(*r)
+        for r in run(
+            ast.parse(s),
+            list(tokenize.generate_tokens(StringIO(s).readline)),
+        )
+    }
+
+
+@pytest.mark.parametrize(
+    "source",
+    (
+        pytest.param(
+            "conftest = 'foo'",
+            id="set conftest",
+        ),
+    ),
+)
+def test_noop(source):
+    assert not results(source)
+
+
+@pytest.mark.parametrize(
+    "source, expected",
+    (
+        pytest.param(
+            "import conftest",
+            "1:0: PDF010 import from 'conftest' found",
+            id="import from conftest",
+        ),
```

Comment:
nice! Can you also add a testcase which has `from conftest import foo`? (this one may not pass... :flushed: )

Reply:
this one likely passes, but `from pandas.conftest import foo` likely won't - can you include that one as well? If you know how to fix the source code to make it pass, then even better :+1:

Reply:
yes for sure!

Reply:
`from pandas.conftest import foo` didn't pass -- should i open an issue for that case separately?

Reply:
sure - let's take this one as it is, and if you're interested in making a separate PR to make `from pandas.conftest import foo` pass as well, then even better! (else I'll do it later)

Reply:
yeh that's what i was getting at 🤣  i wanted to work on making tests pass for that case

Reply:
:laughing: cool - I went and opened a new issue anyway, #16